packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -292,9 +292,9 @@ class ThemeData with Diagnosticable {
     bool? useMaterial3,
     VisualDensity? visualDensity,
     // COLOR
-    // [colorScheme] is the preferred way to configure colors. The other color
-    // properties (as well as primaryColorBrightness, and primarySwatch)
-    // will gradually be phased out, see https://github.com/flutter/flutter/issues/91772.
+    /// [colorScheme] or [colorSchemeSeed] are the preferred way to configure
```

Comment:
I don't think making this a dartdoc here (`///`) will surface it in the public docs since this is just the parameter. Can you add this to the docs for ThemeData.brightness?

Reply:
Right, but it's just to get click-to-definition working in the editors